tests/kernel/fatal/src/main.c

```diff
@@ -162,7 +162,8 @@ void main(void)
 	}
 
 	TC_PRINT("test alt thread 3: initiate kernel panic\n");
-	k_thread_create(&alt_thread, alt_stack, sizeof(alt_stack),
+	k_thread_create(&alt_thread, alt_stack,
```

Comment:
Digression: Seems like this is going to be a persistent error with the new API.  If the stack_size argument to k_thread_create() must always be the result of a K_THREAD_STACK_SIZEOF() macro, it probably shouldn't be a size_t named "stack_size" and documented as "stack size in bytes".

A little work could allow the compiler to typecheck this by having a "k_static_stack" type or somesuch.